frontend/src/scenes/feature-flags/featureFlagsLogic.ts

```diff
@@ -60,6 +71,24 @@ export const featureFlagsLogic = kea<featureFlagsLogicType>({
                 },
             ],
         ],
+        uniqueCreators: [
+            (selectors) => [selectors.featureFlags],
+            (featureFlags) => {
+                const creators = {}
+                featureFlags.forEach((flag: FeatureFlagType) => {
```

Comment:
I don't think you need to type `: FeatureFlagType` here. It should come directly from the input selector. 

I'd personally also write this as a simple `for (const flag of featureFlags) {` instead of `forEach`. Mostly because it's a [bit](https://leanylabs.com/blog/js-forEach-map-reduce-vs-for-for_of/) faster and I think also cleaner with scoping... The real world speed difference for one loop really negligible, so treat this as a nit.